app/frontend/src/features/profile/edit/EditProfile.tsx

```diff
@@ -206,6 +206,7 @@ export default function EditProfileForm() {
             />
           </form>
           <Controller
+            defaultValue=""
```

Comment:
This is to stop the `react-hook-form` warning, even though we don't actually use the value of this field anyway...